src/Template/rector.php

```diff
@@ -127,7 +127,10 @@
     $rectorConfig->rule(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
     $rectorConfig
         ->ruleWithConfiguration(TypedPropertyFromAssignsRector::class, [
-            // Set to false if you use in libraries, or it does create breaking changes.
+            /**
+             * The INLINE_PUBLIC value is default to false to avoid BC break, if you use for libraries and want to preserve BC break, you don't need to configure it, as it included in LevelSetList::UP_TO_PHP_74
+             * Set to true for projects that allow BC break
+             */
             TypedPropertyFromAssignsRector::INLINE_PUBLIC => true,
```

Comment:
I have applied this setting to an App, but many files that extend framework classes were broken because not all properties are typed.

Realistically, it seems to be difficult to use this setting with true.

Reply:
Yes, that's `true` is usefull when need to be applied for whole project.

Probably next improvement to verify `vendor/` location like in `ParentClassMethodTypeOverrideGuard`

https://github.com/rectorphp/rector-src/blob/8354587b777450cb9f2e3bd48cf750d81b78cf05/packages/VendorLocker/ParentClassMethodTypeOverrideGuard.php#L61-L84